core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/models.scala

```diff
@@ -9,24 +9,30 @@ import cats.mtl.Ask
 import io.opencensus.trace.Span
 import org.broadinstitute.dsde.workbench.model.TraceId
 
-final case class AppContext(traceId: TraceId, now: Instant, span: Option[Span] = None) {
+/**
+ * @param traceId Unique identifier to connect logs for the same request or transaction together
+ * @param now
+ * @param requestUri API request if this is context for an API request
+ * @param span span for a request if this is context for an API request
+ */
+final case class AppContext(traceId: TraceId, now: Instant, requestUri: String = "", span: Option[Span] = None) {
```

Comment:
normally I wouldn't add default value. But here, this is just informational, so I think it's okay